src/nylas.ts

```diff
@@ -183,6 +183,9 @@ class Nylas {
     if (options.provider != null) {
       url += `&provider=${options.provider}`;
     }
+    if(options.redirectOnError){
+      url += "&redirect_on_error=true"
+    }
```

Comment:
This block of code fails our `prettier` guideline check, the below suggestion should work!

```suggestion
    if (options.redirectOnError) {
      url += '&redirect_on_error=true';
    }
```
```